typescript/commands/build.py

```diff
@@ -0,0 +1,40 @@
+import sublime_plugin
+import sublime
+import os
+from ..libs.global_vars import IS_ST2
+
+
+class TypescriptBuildCommand(sublime_plugin.WindowCommand):
+    def run(self):
+        file_name = self.window.active_view().file_name()
+        directory = os.path.dirname(file_name)
+        if "tsconfig.json" in os.listdir(directory):
```

Comment:
Does this detect a tsconfig.json in any parent dir?  That's what the server does, and uses that for the language service settings.  If the server is already looking for and loading this file, maybe better to just ask the server if it knows about this file and where, than repeat logic to locate it.